src/App.svelte

```diff
@@ -167,7 +167,7 @@
     {/if}
   </header>
   <main>
-    <div class="mzp-l-content">
+    <div class="mzp-l-content" style="padding-top: 15px">
```

Comment:
they changed the container padding in the latest version of Protocol, so we need to do inline styling here to override it or we'd have a lot of extra padding, like the screenshot below.

<img width="888" alt="Screen Shot 2021-05-12 at 3 50 06 PM" src="https://user-images.githubusercontent.com/28797553/118053703-dc921100-b339-11eb-9a8e-53f1deb93a29.png">

With SvelteKit I'm only importing what we need to make it easier for customization like this (and less inline styling).

Reply:
Awesome, sounds like a sensible compromise for now!